ldap_jwt_auth/auth/authentication.py

```diff
@@ -33,6 +38,9 @@ def authenticate(self, user_credentials: UserCredentialsPostRequestSchema) -> No
         if not username or not password:
             raise InvalidCredentialsError("Empty username or password")
 
+        if not self.is_user_active(username):
```

Comment:
I think the comment above could do with updating.

Reply:
Addressed in https://github.com/ral-facilities/ldap-jwt-auth/pull/22/commits/6a98d4ced706fe884584f1da66dbb36f95b03225